frontend/javascripts/oxalis/model/reducers/skeletontracing_reducer_helpers.ts

```diff
@@ -143,6 +144,7 @@ export function createNode(
     .getOrElse(Constants.DEFAULT_NODE_RADIUS);
   // Find new node id by increasing the max node id.
   const nextNewId = skeletonTracing.cachedMaxNodeId + 1;
+  const position = V3.floor(positionFloat);
```

Comment:
This could be a `round` operation, too, but when creating a node at z=1.6, this would change the behavior. Until now, that node would be stored at z=1. Rounding would store it at z=2. Depending on the clipping distance, this could be extra weird. For moving a node, this issue isn't relevant, I think, because the movement is 2D.